Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -232,7 +232,7 @@ public HumanoidCharacterProfile WithAntagPreference(string antagId, bool pref)
         }
 
         public string Summary =>
-            $"{Name}, {Age} years old {Sex.ToString().ToLower()} human.";
+            $"{Name}, {Age} years old human. Their gender is {Sex.ToString().ToLower()}.";
```

Comment:
1. Sex is not same as gender. Just, don't. *Especially* when you have it shown as "Sex" in the character creation, have it called like this everywhere, and then do *this*. I am assuming that so far (ignoring this line) that sex meant more the general appearance of character? Which is also not tied to biological bits, just so you know. 
2. There is a pronouns system within Robust Toolbox specially for adding them into text.
3. "IDK" is not something to resolve a "How does it work with localization" problem.